src/api/app/controllers/webui/project_controller.rb

```diff
@@ -66,14 +66,14 @@ def autocomplete_incidents
   def autocomplete_packages
     @project = Project.find_by(name: params[:project])
     if @project
-      render json: @project.packages.autocomplete(params[:term]).limit(50).pluck(:name)
+      render json: @project.packages.autocomplete(params[:term]).limit(50).order(:name).pluck(:name)
```

Comment:
move the order to the scope...

Reply:
If you ask that nicely... :wink:

Reply:
Can you please move it? This is not its place. :bowtie:

Reply:
Already did, but seems I overlooked this one since it's [defined](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/project.rb#L33) within `has_many :packages` definition of the project model:unamused: